tests/neg/i17192.5.scala

```diff
@@ -0,0 +1,12 @@
+class Ifce[BT <: Boolean]:
```

Comment:
Is there a reason this file has `.5` in its name?

Reply:
I was the minimisation from the 5th comment.  Just in case I end up using multiple separate versions.